src/ansys/geometry/core/plotting/plotter.py

```diff
@@ -379,7 +384,11 @@ def add(
             Mapping between the ~pyvista.Actor and the PyAnsys Geometry object.
         """
         logger.debug(f"Adding object type {type(object)} to the PyVista plotter")
-
+        if filter:
+            if hasattr(object, "name"):
+                if not re.search(filter, object.name):
+                    logger.info(f"Name {object.name} not found in regex {filter}.")
```

Comment:
Use debug rather than info. Too verbose otherwise
```suggestion
                    logger.debug(f"Name {object.name} not found in regex {filter}.")
```